test/helpers/reactjs_helper_test.rb

```diff
@@ -3,8 +3,8 @@
 class ReactjsHelperTest < ActionView::TestCase
   include ReactjsHelper
 
-  def webpack_asset_paths(bundle_name, opts)
-    ["<script src=\"https://foreman.example.com:3808/webpack/#{bundle_name}.js\"></script>"]
+  def javascript_pack_tag(bundle_name, opts)
```

Comment:
does this works on dev? i thought it should use the different port to access the dev-server?

Reply:
It's a test file! the meaning is only to cancel the default behavior of webpacker so it won't actually search inside the `manifest.json` and load an entry which doesn't really exist.

It won't even add it to the html.